### PR TITLE
OCPBUGS-35480:updates prerequisites for OVN-K migration

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OCPBUGS-35840

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by anuragthehatter** [comment]
cc @weliang1 @zhaozhanqi PTAL

**Comment by weliang1** [comment]
@JoeAldinger Do we have a bug for this new statement? 
`Before migration to OVN-Kubernetes, set all timeouts for webhooks to 3 seconds or remove them.`

**Comment by JoeAldinger** [comment]
> @JoeAldinger Do we have a bug for this new statement? `Before migration to OVN-Kubernetes, set all timeouts for webhooks to 3 seconds or remove them.`

Yes, it is linked above.

File: modules/nw-ovn-kubernetes-migration.adoc [hunk 1]
```diff
@@ -25,6 +25,7 @@ Perform the migration only when an interruption in service is acceptable.
 * A reboot can be triggered manually for each node.
 * The cluster is in a known good state, without any errors.
 * Before migration to OVN-Kubernetes, a security group rule must be in place to allow UDP packets on port `6081` for all nodes on all cloud platforms.
+* Before migration to OVN-Kubernetes, set all timeouts for webhooks to 3 seconds or remove them.
```
Comment (dfitzmau) [inline modules/nw-ovn-kubernetes-migration.adoc:28]:
```suggestion
* Before you migrate your cluster to use the OVN-Kubernetes plugin, either remove timeouts for webhooks or set the timeouts to `3` seconds.
```

**Review by dfitzmau — COMMENTED** [review]
/remove-label peer-review-in-progress

/label peer-review-done

**Pull request merged**